File: LinkedList/src/main/java/org/mainProgramme/Queue/Queue.java
package org.mainProgramme.Queue;

public class Queue<T> {
    Deque<T> store = new Deque<>();

    //Enqueue an item
    public void Enqueue(T value) { store.EnqueueTail(value); }

    //Dequeue an item
    public T Dequeue() { return store.DequeueHead(); }

    //Peek an item(head)
    public T peek() { return store.peekHead(); }

}

File: Graph/Queue.py
from collections import deque

class Queue:
    def __init__(self):
        self.queue = deque()
    def enqueue(self, node):
        self.queue.append(node)
    def dequeue(self):
        a = self.peek()
        self.queue.popleft()
        return a
    def peek(self):
        return self.queue[0]
    def isEmpty(self) -> bool:
        return len(self.queue) < 1